src/core/reducers/user.js

```diff
@@ -30,6 +32,17 @@ export const isAuthenticated = (state: { user: UserStateType }) => {
   return !!state.user.id;
 };
 
+export const selectDisplayName = (state: { user: UserStateType }) => {
+  const displayName = state.user.displayName;
+  if (displayName && displayName !== '') {
+    return displayName;
+  }
+
+  // We fallback to the username if no display name has been defined by the
```

Comment:
Weird, I thought `display_name` would be `null` if the user hadn't picked one. I'd also sort of expect it to fall back to `username`.

I guess there's nothing wrong with this approach but I'm surprised we'd ever get an empty string like that back from the API.